server/src-lib/Hasura/RQL/DML/Returning.hs

```diff
@@ -135,7 +135,7 @@ mkMutationOutputExp qt allCols preCalAffRows cte mutOutput strfyNum =
     mutationResultAlias = Iden $ snakeCaseQualObject qt <> "__mutation_result_alias"
     allColumnsAlias = Iden $ snakeCaseQualObject qt <> "__all_columns_alias"
     allColumnsSelect = S.CTESelect $ S.mkSelect
-                       { S.selExtr = map S.mkExtr $ map pgiColumn $ sortCols allCols
+                       { S.selExtr = map (S.mkExtr . pgiColumn) (sortCols allCols)
```

Comment:
Why not `map (S.mkExtr . pgiColumn) $ sortCols allCols`?

Reply:
Ah, well hlint doesn't have a clear preference for for `$` or `()`. From our point of view, I agree that indeed perhaps that reads more nicely. So I'll implement this suggestion.